src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/config/BpdmCommonAutoConfiguration.kt

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.eclipse.tractusx.bpdmcertificatemanagement.config
+
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+@ComponentScan
+@ConfigurationPropertiesScan
+class BpdmCommonAutoConfiguration
```

Comment:
Here again, should be removed

Reply:
Renamed, followed approach from bpdm.

Reply:
I didn't mark it because of the name but because I don't know why it is there in the first place. The SpringBootApplication annotation already includes componentscan and is also a good candidate for the the configurationpropertiesscan annotation. Autoconfigurations are for spring libraries that should be included in spring projects. This is an application that does not need autoconfiguration as far as I can see

Reply:
I thought in different way, removed autoconfiguration.